packages/app/src/utils/API/nominationsAPI.js

```diff
@@ -7,6 +7,11 @@ const nominationsAPI = {
   fetchNomination: function (id) {
     return axios.get(`/nominations/${id}`);
   },
+  updateNomination: function (id, w) {
```

Comment:
What is `w` supposed to represent here? Can we use a more descriptive variable name?